docs/user_guide.rst

```diff
@@ -0,0 +1,90 @@
+
+
+.. _Folding@Home: https://foldingathome.org
+
+
+************
+Installation
+************
+
+Create a conda environment with the ``alchemiscale`` client using, e.g. `micromamba`_::
+
+    $ micromamba create -n alchemiscale-client -c conda-forge alchemiscale-client
+
+
+Once installed, activate the environment and install additional dependencies::
+
+    $ micromamba activate alchemiscale-client
+    $ micromamba install feflow>=0.1.2 plyvel
+
+Finally, install ``alchemiscale-fah`` where ``<release-tag>`` corresponds to the GitHub tag from the release of **alchemiscale-fah** deployed on the target ``alchemiscale`` instance::
+
+    $ pip install git+https://github.com/OpenFreeEnergy/alchemiscale-fah.git@<release-tag>
+
+You may wish to install other packages into this environment, such as ``jupyterlab``.
+
+.. _micromamba: https://github.com/mamba-org/micromamba-releases
+
+
+******************************************************************
+Creating an AlchemicalNetwork using a Folding\@Home-based Protocol
+******************************************************************
+
+**alchemiscale-fah** features :external+gufe:py:class:`~gufe.protocols.Protocol`\s derived from packages such as ``feflow``.
+These take advantage of most of the components of these :external+gufe:py:class:`~gufe.protocols.Protocol`\s, but perform the compute-intensive simulation workloads using Folding\@Home instead.
+
+To create an :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`, review this notebook and apply the same approach to your systems of interest and using a Folding\@Home-based :external+gufe:py:class:`~gufe.protocols.Protocol` like those listed below: `Preparing AlchemicalNetworks.ipynb`_
+
+Currently, the following :external+gufe:py:class:`~gufe.protocols.Protocol`\s are available to users:
+
+* :py:class:`alchemiscale_fah.protocols.feflow.nonequilbrium_cycling.FahNonEquilibriumCyclingProtocol`
+
+Try each one out with default options for a start.
+Below are notes on settings you may find more optimal for each, however.
+
+.. _Preparing AlchemicalNetworks.ipynb: https://github.com/OpenFreeEnergy/ExampleNotebooks/blob/main/networks/Preparing%20AlchemicalNetworks.ipynb
+
+
+``FahNonEquilibriumCyclingProtocol`` usage notes
+================================================
+
+For production use of this protocol, we recommend the default settings::
+
+    >>> from alchemiscale_fah.protocols.feflow import FahNonEquilibriumCyclingProtocol
+
+    >>> settings = NonEquilibriumCyclingProtocol.default_settings()
+
+These default settings will perform non-equilibrium cycling with a total simulation time of 40 ns for each cycle, starting with 10 ns of equilibrium sampling in state A, 10 ns of nonequilibrium sampling from state A to B, 10 ns of equilibrium sampling in state B, and finally 10 ns of nonequilibrium sampling from state B to A.
```

Comment:
> 40 ns for each cycle

Is this correct? Assuming "equilibrium_steps" and "nonequilibrium_steps" are in units of timstep, then 250,000 should be 1 ns not 10 ns?

Reply:
You're exactly right; I was off by 1 in my factors of 10 checking this. Thank you!